components/mpas-ocean/src/shared/mpas_ocn_time_average_coupled.F

```diff
@@ -311,6 +343,48 @@ subroutine ocn_time_average_coupled_accumulate(statePool, forcingPool, timeLevel
            !$omp end parallel
         end if
 
+        ! Accumulate polar fields if necessary
+        if(trim(config_land_ice_flux_mode) == 'standalone' .or. 'data') then
+           call mpas_pool_get_array(forcingPool, 'avgLandIceFreshwaterFlux', avgLandIceFreshwaterFlux)
+
+           if(trim(config_land_ice_flux_mode) == 'standalone') then
+              call mpas_pool_get_array(forcingPool, 'landIceFreshwaterFlux', landIceFreshwaterFlux)
+           endif
+
+          ! load data melt rates if used
```

Comment:
this didn't work, stays 0.

Reply:
You should be able to use `landIceFreshwaterFlux` for both `standalone` and `data`, since one is just copied to the other:
https://github.com/E3SM-Project/E3SM/blob/104a2027d9ad34fbd9198422641ad47f19db65c0/components/mpas-ocean/src/shared/mpas_ocn_surface_land_ice_fluxes.F#L300
But that doesn't explain why you're getting zero.